src/client/components/vision2/image_view/stories/image_view.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import { storiesOf } from '@storybook/react'
+import { observable } from 'mobx'
+import { computed } from 'mobx'
+import { action } from 'mobx'
+import { Observer } from 'mobx-react'
+import React from 'react'
+
+import { fullscreen } from '../../../storybook/fullscreen'
+import { ImageFormat } from '../../image'
+import { Image } from '../../image'
+import { ImageView } from '../view'
+
+import jpegUrl from './images/image.jpg'
+import jprgUrl from './images/image.jprg.jpg'
+import rggbUrl from './images/image.rggb.bin'
+
+const createModel = () => observable<{ image: Image | undefined}>({ image: undefined })
+
+storiesOf('components.vision2.image_view', module)
+  .addDecorator(fullscreen)
+  .add('JPEG', () => {
+    const model = createModel()
+    loadImageElement(jpegUrl, ImageFormat.JPEG).then(action((image: Image) => model.image = image))
+    return <Observer>{() => <>
+      {model.image ? <ImageView image={model.image}/> : null}
+    </>}</Observer>
+  })
+  .add('JPRG', () => {
+    const model = createModel()
+    loadImageElement(jprgUrl, ImageFormat.JPRG).then(action((image: Image) => model.image = image))
+    return <Observer>{() => <>
+      {model.image ? <ImageView image={model.image}/> : null}
+    </>}</Observer>
+  })
+  .add('RGGB', () => {
+    const model = createModel()
+    loadImageData(rggbUrl, 1280, 1024, ImageFormat.RGGB).then(action((image: Image) => model.image = image))
```

Comment:
This is a pretty big file to be committing to the repo, maybe instead we should generate a bayer pattern image on the fly from a regular image. So converting
`RGB` `RGB` `RGB` `RGB`
`RGB` `RGB` `RGB` `RGB`
to
`R` `G` `R` `G`
`G` `B` `G` `B`

for an `RGGB` bayer pattern